### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by dominikl**
The README.md is a bit out of date. This PR updates some information. Also links to the release page, as we've got the package there for download and easy installation without the need of using the `install.R` script (and having all its dependencies installed).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,76 @@
...
+
+Alternatively: In R console run `install.packages('romero.gateway_x.x.x.tar.gz', type = 'source')`
+
+See [Tips for Unix users](#tips-for-unix-users) section if you're running into difficulties.
 
 ## Usage
+
 * Like any other R package load the package ```library(romero.gateway)```
 * Try some examples from the [examples directory](examples)
+* Another potential source for examples is the unit tests directory: [tests/testthat](tests/testthat)
```
Comment (mtbc) [inline README.md]: source *of*

File: README.md [hunk 2]
```diff
@@ -54,19 +79,25 @@ the test server
...
-#### Additional dependencies
+### Additional dependencies
+
 In order to build/install some necessary R packages, additional system libraries may
 have to be installed first. E.g. the R packages `httr` and `xml2` need the development libraries for
-`curl` and `xml2`, so for example on a Debian system you most likely have to install `libcurl4-dev`
+`curl` and `xml2`, so for example on a Debian system you probably have to install `libcurl4-dev`
 and `libxml2-dev` first.
+If you can't install an R package due to compile errors, most likely you are missing a development 
+package of some dependent library on your system.
```
Comment (mtbc) [inline README.md]: Good tip. :+1:

File: README.md [hunk 3]
```diff
@@ -1,51 +1,76 @@
...
-* Build the package ```devtools::build()```
-* Install the romero.gateway package into your local R repository
-  ```
-  install.packages([PATH TO romero.gateway_x.y.z.tar.gz], repos = NULL, type="source")
-  ```
+## Install the romero.gateway R package
+
+### Windows
+
+Download the zip package from the [Release page](https://github.com/ome/rOMERO-gateway/releases)
```
Comment (jburel) [inline README.md]: the zip package matching your omero server

File: README.md [hunk 4]
```diff
@@ -1,51 +1,76 @@
...
+
+In RStudio "Install Packages" use option "Install from:" "Package Archive File" and select the romero.gateway zip file.
+
+Alternatively: In R console run `install.packages('romero.gateway_x.x.x.zip')`
+
+### Unix / OSX
+
+Download the tar.gz package from the [Release page](https://github.com/ome/rOMERO-gateway/releases)
```
Comment (jburel) [inline README.md]: same here

File: README.md [hunk 5]
```diff
@@ -1,51 +1,76 @@
...
+
+In RStudio "Install Packages" use option "Install from:" "Package Archive File" and select the romero.gateway tar.gz file.
+
+Alternatively: In R console run `install.packages('romero.gateway_x.x.x.tar.gz', type = 'source')`
+
+See [Tips for Unix users](#tips-for-unix-users) section if you're running into difficulties.
```
Comment (jburel) [inline README.md]: you are

File: README.md [hunk 6]
```diff
@@ -1,51 +1,76 @@
...
+  Rscript install.R
+```
+
+You can specify a particular branch or version to build/install or perform a local build of the cloned repository. Run `Rscript install.R --help` to see more details.
+
+### Manually
+
+* Download the dependencies
+  * Using Maven: Run `mvn install`
+  * Alternative: Create `inst/java` directory. Download OMERO.insight client. Extract the zip file. Copy all files within libs directory into the previously created `inst/java` directory
```
Comment (jburel) [inline README.md]: Copy all the files within the `libs` directory

**Pull request merged**